tests/util/caches/test_sync_responsecache.py

```diff
@@ -0,0 +1,134 @@
+from synapse.util.caches.sync_response_cache import SyncResponseCache
+
+from tests.server import get_clock
+from tests.unittest import TestCase
+
+# A random callback that returns an object
+CALLBACK = lambda: OBJ
+
+# An object, can be equalized to itself
+OBJ = {0}
```

Comment:
All that matters is that i have something with an identifier that `x == x`, so i can easily boilerplate it.